src/templates/sitemap.xml

```diff
@@ -12,7 +12,7 @@
     </url>
 
     <url>
-        <loc>https://almanac.httparchive.org/en/2019/accessibility_statement</loc>
+        <loc>https://almanac.httparchive.org/en/2019/accessibility-statement</loc>
```

Comment:
I think this still has an issue pointing to 2019 when the live URL is https://almanac.httparchive.org/en/accessibility-statement